pyproject.toml

```diff
@@ -81,10 +86,6 @@ strict_equality = true
 disallow_untyped_defs = true
 explicit_package_bases = true
 
-[[tool.mypy.overrides]]
-module = ["beamer.tests.*"]
-disallow_untyped_defs = false
-
```

Comment:
It seems we have this rebase artifact again. Could you please remove this and the related `tool.mypy.overrides` hunk above so your changes don't affect those?

Reply:
I would really like to understand how this ends up there.